### ISSUE TITLE
Need help running codes

### ISSUE THREAD

**Issue by lczong** [MoonLBH/Apindel#2]
Hi there,

Thank you for this work! I'm new to this project and I'm having trouble running the codes. I don't know what arguments I should pass to `model_creation/main.py` to run it.

Could someone please provide me with instructions on running the codes successfully? It would be really helpful if you could also include how to place the data in the right place. For example, what are the "result.txt" and "result1.txt" in #line347 `a=rawdata("result.txt","result1.txt")` in `data_compilation/compile_forecast.py`

Thank you in advance for your help!

Lucas

**Comment by Oh-Shmily** [comment]
I am also very troubled because he did not update the final version of the code. I guess result.txt and result1.txt are both data from the "FORECAST" dataset, and I am working hard to replicate their work

**Comment by FAhtisham** [comment]
Any success in reproducing the results?

**Comment by Oh-Shmily** [comment]
> Any success in 复制 the results?

Yes, but I almost rewrote the data preprocessing part, use the forecast and sprout data

**Comment by FAhtisham** [comment]
> > Any success in 复制 the results?
> 
> Yes, but I almost rewrote the data preprocessing part, use the forecast and sprout data

Could you please share that part ? Particularly, I am not sure which one is the result.txt and result1.txt file
